source/wp-content/themes/wporg-developer-2023/patterns/front-page-content.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Title: Front Page Content
+ * Slug: wporg-developer-2023/front-page-content
+ * Inserter: no
+ */
+
+?>
+
+<!-- wp:group {"className":"has-white-color has-charcoal-2-background-color has-text-color has-background has-link-color","layout":{"type":"constrained"}} -->
+<div class="wp-block-group has-white-color has-charcoal-2-background-color has-text-color has-background has-link-color"><!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|60"}}},"layout":{"type":"constrained","contentSize":"1344px"}} -->
+<div class="wp-block-group alignfull" style="padding-top:var(--wp--preset--spacing--60);padding-bottom:var(--wp--preset--spacing--60)"><!-- wp:image {"align":"wide","id":7564,"sizeSlug":"full","linkDestination":"none","className":"wporg-hero-graphic"} -->
+<figure class="wp-block-image alignwide size-full wporg-hero-graphic"><img src="https://s.w.org/wp-content/themes/wporg-developer-2023/images/developers.svg?v=20230116" alt="<?php esc_attr_e( 'WordPress Developer Resources', 'wporg' ); ?>" class="wp-image-7564"/></figure>
+<!-- /wp:image --></div>
+<!-- /wp:group -->
+
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"bottom":"var:preset|spacing|60","right":"var:preset|spacing|edge-space","left":"var:preset|spacing|edge-space"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignfull" style="padding-right:var(--wp--preset--spacing--edge-space);padding-bottom:var(--wp--preset--spacing--60);padding-left:var(--wp--preset--spacing--edge-space)"><!-- wp:group {"align":"wide","layout":{"type":"default"}} -->
+<div class="wp-block-group alignwide"><!-- wp:search {"label":"<?php esc_attr_e( 'Search', 'wporg' ); ?>","showLabel":false,"placeholder":"<?php esc_attr_e( 'Search in the code reference...', 'wporg' ); ?>","width":432,"widthUnit":"px","buttonText":"Search","buttonPosition":"button-inside","buttonUseIcon":true} /--></div>
+<!-- /wp:group --></div>
+<!-- /wp:group --></div>
+<!-- /wp:group -->
+
+<!-- wp:group {"backgroundColor":"blueberry-4","style":{"spacing":{"padding":{"top":"var:preset|spacing|10","bottom":"var:preset|spacing|10"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group has-blueberry-4-background-color has-background" style="padding-top:var(--wp--preset--spacing--10);padding-bottom:var(--wp--preset--spacing--10)"><!-- wp:paragraph {"align":"center","fontSize":"extra-small"}  -->
+<p class="has-text-align-center has-extra-small-font-size">
+<?php printf(
+	/* translators: %s: WordPress version number link. %s WordPress version number  */
+	__( 'See <a href="%s">what has changed</a> in the WordPress %s API.'),
+	do_shortcode( 'wordpress_version_link' ),
+	do_shortcode( 'wordpress_version') ); ?>
```

Comment:
Getting some local lint errors:

![Screen Shot 2023-01-16 at 1 26 32 PM](https://user-images.githubusercontent.com/1017872/212575808-4dda92da-26f2-4f06-afac-9b4561aec164.jpg)

And the shortcodes aren't returning for me; I see this:

![Screen Shot 2023-01-16 at 1 29 14 PM](https://user-images.githubusercontent.com/1017872/212575834-cde3a501-4515-4c70-a671-b2aa241324fc.jpg)

Happy to try to debug that if you can't repro

Reply:
Interestingly, there were 2 issues:
1. Using `do_shortcode`, I needed to wrap with `[]`.
2. Using `do_shortcodes` like this breaks context and the underlying functions don't work. No idea why.

Used `wp_kses_post` instead.

Reply:
These last minute linter fixes are killing me :)

Reply:
Working for me now